capstone-frontend/src/pages/UserProfile.jsx

```diff
@@ -35,6 +37,20 @@ function UserProfile() {
     fetchUserData();
   }, []);
 
+  useEffect(() => {
```

Comment:
Can you use SWR for this? Might simplify?

Reply:
I found using regular fetch to be easier to me, I transitioned back to fetch as I attempted using SWR but had difficulty using it.